senza.yaml

```diff
@@ -0,0 +1,162 @@
+    - TokeninfoUrl:
+        Description: "TokeninfoUrl"
+    - ApplicationId:
+        Description: "Application id"
+        Default: "nakadi"
+    - MintBucket:
+        Description: "Name of the mint bucket to use for this application"
+    - ScalyrAccountKey:
+        Description: "Key for Skalyr logging (optional)"
+        Default: ""
+    - InstanceType:
+        Description: "The EC2 Instance type that is used for the Nakadi instances (optional, default: t2.micro)"
+        Default: "t2.micro"
+    - InstancesMinumum:
+        Description: "The minimum number of instances of the AutoScalingGroup (optional, default: 2)"
+        Default: 2
+    - InstancesMaximum:
+        Description: "The maximum number of instances of the AutoScalingGroup (optional, default: 5)"
+        Default: 5
+    - ZookeeperKafkaNamespace:
+        Description: "Namespace in Zookeeper used by Buku (Kafka) (optional)"
+        Default: ""
+    - ExhibitorBrokers:
+        Description: "List of exhibitor brokers (comma separated)"
+    - ExhibitorPort:
+        Description: "Exhibitor port (ptional, default 8181)"
+        Default: "8181"
+    - LimitTrafficToCidr:
+        Description: "Allows request to the loadbalancer of Nakadi to only this ip address range given in cidr notation (optional, default: 0.0.0.0/0)"
+        Default: "0.0.0.0/0"
+
+SenzaComponents:
+
+  - Configuration:
+      Type: Senza::StupsAutoConfiguration # auto-detect network setup
+
+  - AppServer:
+      Type: Senza::TaupageAutoScalingGroup
+      InstanceType: "{{Arguments.InstanceType}}"
+      SecurityGroups:
+        - "Fn::GetAtt":
+          - "NakadiSecurityGroup"
+          - "GroupId"
+      IamInstanceProfile:
+        Ref: NakadiIamInstanceProfile
+      ElasticLoadBalancer: AppLoadBalancer
+      AssociatePublicIpAddress: false
+      TaupageConfig:
+        application_id: "{{Arguments.ApplicationId}}"
+        application_version: "{{SenzaInfo.StackVersion}}"
+        mint_bucket: "{{Arguments.MintBucket}}"
+        runtime: Docker
+        source: "{{Arguments.DockerImage}}"
+        health_check_path: /health
+        ports:
+          8080: 8080
+        scalyr_account_key: "{{Arguments.ScalyrAccountKey}}"
+        environment:
+          HTTP_TOKENINFO_URL: "{{Arguments.TokeninfoUrl}}"
+          OAUTH2_ACCESS_TOKEN_URL: "{{Arguments.AccessTokenUrl}}"
+          ARUHA_PUSH_OAUTH_ACTIVE: "true"
+          ARUHA_PULL_OAUTH_ACTIVE: "true"
+          ARUHA_SUBSCRIPTION_MODE: "zookeeper"
+          STREAM_HEARTBEAT_PERIOD_MS: "5000"
+          KAFKA_CLIENTS_INIT_POOL_SIZE: "4"
+          KAFKA_CLIENTS_MAX_POOL_SIZE: "16"
+          NAKADI_ZOOKEEPER_KAFKANAMESPACE:    "{{Arguments.ZookeeperKafkaNamespace}}"
+          NAKADI_ZOOKEEPER_EXHIBITOR_BROKERS: "{{Arguments.ExhibitorBrokers}}"
+          NAKADI_ZOOKEEPER_EXHIBITOR_PORT:    "{{Arguments.ExhibitorPort}}"
+      AutoScaling:
+        Minimum: "{{Arguments.InstancesMinumum}}"
```

Comment:
No, unfortunately it wont work, because `{}` -- declares dict in yaml...
As a first step senza loads yaml file (without any manipulations with templates) and only afterwards it traverse the tree and replaces parameters with some values.